Objects/Sensor.py

```diff
@@ -61,17 +88,45 @@ def __init__(self, monitor_num, box=None):
             
     def grab_data(self):
         img = self.sct.grab(self.box)
-        return np.array(img)
+        return np.array(img)[...,:3].astype(float)
```

Comment:
It would be good to add a comment on why filtering the number of channels is necessary